package.json

```
@@ -38,9 +38,9 @@
     "@babel/plugin-transform-runtime": "^7.19.6",
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
-    "@decidim/dev": "0.27.1",
-    "@decidim/eslint-config": "0.27.1",
-    "@decidim/stylelint-config": "0.27.1",
+    "@decidim/dev": "0.27.3",
+    "@decidim/eslint-config": "0.27.3",
+    "@decidim/stylelint-config": "0.27.3",
```

```suggestion
    "@decidim/dev": "0.27.4",
    "@decidim/eslint-config": "0.27.4",
    "@decidim/stylelint-config": "0.27.4",
```
we can upgrade to 27.4 (do `npm i`)